scripts/exec.mjs

```diff
@@ -0,0 +1,47 @@
+import {spawn} from 'child_process';
+
+/**
+ * @param {string} str
+ */
+function trimNewline(str) {
+  return str.endsWith('\n') ? str.slice(0, -1) : str;
+}
+
+/**
+ * @param {string} command
+ * @param {readonly string[]} [args]
+ * @returns {Promise<string>}
+ */
+export function execute(command, args = []) {
```

Comment:
Could we add a package to do this type of thing? I feel like it's a bit out of our scope